lib/run.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+require('dotenv').config();
+
+if (process.env.NEWRELIC_KEY) {
+  require('newrelic'); // eslint-disable-line global-require
+}
+
+const { findPrivateKey } = require('probot/lib/private-key');
+const createProbot = require('probot');
+const app = require('.');
+
+const probot = createProbot({
+  id: process.env.APP_ID,
+  secret: process.env.WEBHOOK_SECRET,
+  cert: findPrivateKey(),
+  port: process.env.PORT || 3000,
+  webhookPath: '/github/events',
+  webhookProxy: process.env.WEBHOOK_PROXY_URL,
```

Comment:
Do we need some production specific logic for this?

Reply:
Nope. Probot will do nothing if this is blank. I don't think we're actually using smee due to the slack side, so we should probably just skip this